spec/forms/applicants/basic_details_form_spec.rb

```diff
@@ -4,7 +4,7 @@
   subject { described_class.new(params) }
 
   describe ".model_name" do
-    it 'should be "Applicant"' do
+    it 'is an "Applicant"' do
```

Comment:
<pedantic>Shouldn't this be 'Is "Applicant"'</pedantic>
We are checking the contents of a string, not the object type?